index.html

```diff
@@ -523,6 +524,116 @@ <h4>Workflow Type</h4>
+          <ul>
+            <li>the <dfn><code>daptm:originalLangs</code></dfn> attribute
+              SHOULD be present on the <code>&lt;tt&gt;</code> element
+              and if present its value MUST NOT be empty.</li>
+            <li>the value of the attribute MUST be a set of space separated language identifiers,
+              each being a language tag as defined by [[BCP47]].</li>
+          </ul>
+          <!-- BCP47's syntax does not permit white space within each language tag -->
+          <div class="exampleInner">
+            <pre class="language-abnf">
+daptm:originalLangs
+  : languageIdentifier (&lt;lwsp&gt; languageIdentifier )*
+
+languageIdentifier  \\ BCP47 language tag
+
+&lt;lwsp&gt;
+  : &lt;whitespace&gt;+
+
+&lt;whitespace&gt;
+  : ( &quot; &quot; | &quot;\t&quot; | &quot;\n&quot; | &quot;\r&quot; )
+            </pre>
+          </div>
+          <aside class="example">
+            <p>A video contains dialogue in Danish and Swedish.
+              Two <a>Original Language Transcripts</a> are prepared,
+              one transcribing the dialogue, as a precursor to hard of hearing subtitles or dubbing,
+              the other transcribing the image, for audio description.</p>
+            <p>The dialogue transcript has its <a>Original languages list</a> set to Danish and Swedish,
+              by setting <code>daptm:originalLangs="da sv"</code> on the <code>&lt;tt&gt;</code> element.
+              The <a>Text</a> objects are each marked up with an appropriate language using <code>xml:lang</code>
+              and <a>Text Language Source</a> using <a><code>daptm:langSrc</code></a> reflecting the language of that part of the dialogue.
+              During the video, a character speaks briefly in Arabic;
+              on that <a>Text</a>, the <a>Text Language Source</a> and language are both set to Arabic,
+              indicating that Arabic is the original language, even though it is not in the document's
+              <a>Original languages list</a>.</p>
+            <p>The video image transcript is being prepared for a Danish audience initially,
+              and has its <a>Original Languages</a> set to Danish only
+              by setting <code>daptm:originalLangs="da"</code> on the <code>&lt;tt&gt;</code> element.
+              The <a>Text</a> objects are each marked up with an appropriate language using <code>xml:lang</code>,
+              and, if representing in-image text, a <a>Text Language Source</a> set to
+              the language of that in-image text using <a><code>daptm:langSrc</code></a>.
+              During the video, a Swedish newspaper headline is visible.
+              This is transcribed with both the language and <a>Text Language Source</a> set to Swedish.
+              Those <a>Text</a> objects representing parts of the scene that have no inherent language
```

Comment:
I think "Any" would be a better alternative word here. Even though the use of "those" here is correct formal English I can see that it could be confusing or misleading and is a good candidate for using simpler language. Will change it.